app/routes/how-can-i-help.community.tsx

```diff
@@ -99,7 +104,7 @@ const Online = () => (
       className="padding-bottom-40"
     >
       <p className="grey default">
-        You can volunteering for projects and attend protests. People are divided regarding whether
+        You can volunteer for projects and attend protests. People are divided regarding whether
```

Comment:
I don't like this copy just because what we're trying to illustrate here something about how communities are grouped, not whether attending protests is important. If we really don't like including the word protests, I would just choose another activity that is common to illustrate our point. This is the only piece of feedback so far I am bullish on.

Reply:
Agreed